Tasks/AzureFileCopy/task.json

```diff
@@ -1,4 +1,4 @@
-﻿{
```

Comment:
@ericsciple Both handler on windows agent and xplat agent handle BOM correctly. Newtonsoft.Json.Parse() and node.require() will discards the BOM properly. the only thing that will break is JavaScript.Json.Parse().